src/App.js

```diff
@@ -37,25 +40,34 @@ const Routes = (props) => {
       <Route path="/" exact component={Home} />
       <Route path="/create-pool" exact component={PoolCreator} />
       <Route path="/explore" exact component={PoolExplorer} />
-      <Route
-        path="/dashboard/:poolAddress"
-        exact
-        component={PoolDashboard}
-      />
+      <Route path="/dashboard/:poolAddress" exact component={PoolDashboard} />
     </Switch>
-  )
-}
+  );
+};
 const App = (props) => {
   const classes = useStyles();
+  const [networkWarningModalOpen, setNetworkWarningModalOpen] = useState(false);
 
+  useEffect(() => {
```

Comment:
I think if you move this effect to the modal, `App` component will be cleaner. What you think?

Reply:
sure